diff -urN htick.orig/src/clean.c htick/src/clean.c
--- htick.orig/src/clean.c	2004-03-28 00:30:04.000000000 +0300
+++ htick/src/clean.c	2007-01-27 11:46:11.000000000 +0300
@@ -115,7 +115,7 @@
             aka = &(config->links[i]->hisAka);
             do
             {
-              if (createOutboundFileNameAka(config->links[i], normal, FLOFILE, aka) == 0)  {
+              if (createOutboundFileNameAka(config->links[i], f_normal, FLOFILE, aka) == 0)  {
                   strcpy(tmpdir, config->links[i]->floFile);
                   sprintf(strrchr(tmpdir, '.'), ".sep");
                   if (direxist(tmpdir)) {
diff -urN htick.orig/src/htickafix.c htick/src/htickafix.c
--- htick.orig/src/htickafix.c	2007-01-10 12:34:26.000000000 +0300
+++ htick/src/htickafix.c	2007-01-27 11:46:11.000000000 +0300
@@ -109,14 +109,14 @@
 		
 		if (link->numOptGrp) {
 			if (grpInArray(echo->group,link->optGrp,link->numOptGrp)) {
-				if (link->export==0) return 3; else rc = 0;
+				if (link->h_export==0) return 3; else rc = 0;
 			}
 		}
 	}
 	
     if (echo->levelread > link->level) return 2;
     /* check for 'access export' for arealink set up by WriteOnly keyword */
-    if (echo->downlinks[i]->export==0) return 3;
+    if (echo->downlinks[i]->h_export==0) return 3;
 
     return rc;
 }
@@ -157,14 +157,14 @@
 
 		if (link->numOptGrp) {
 			if (grpInArray(echo->group,link->optGrp,link->numOptGrp)) {
-				if (link->import==0) return 3; else rc = 0;
+				if (link->h_import==0) return 3; else rc = 0;
 			}
 		}
     }
 	
     if (echo->levelwrite > link->level) return 2;
     /* check for 'access import' for arealink set up by ReadOnly keyword */
-    if (echo->downlinks[i]->import==0) return 3;
+    if (echo->downlinks[i]->h_import==0) return 3;
 
     return rc;
 }
