diff -urN smapi.orig/Makefile smapi/Makefile
--- smapi.orig/Makefile	2003-03-11 17:07:07.000000000 +0300
+++ smapi/Makefile	2007-01-27 16:24:13.000000000 +0300
@@ -24,7 +24,7 @@
 endif
 
 SRC_DIR = src/
-H_DIR   = smapi/
+H_DIR   = smapi
 
 CDEFS=-D$(OSTYPE) $(ADDCDEFS) -I$(H_DIR) -I$(INCDIR)
 
@@ -51,7 +51,7 @@
   else
 $(TARGETDLL).$(VER): $(OBJS)
 	$(CC) -shared -Wl,-soname,$(TARGETDLL).$(VERH) \
-          -o $(TARGETDLL).$(VER) $(OBJS)
+          -L$(LIBDIR) -lhusky -o $(TARGETDLL).$(VER) $(OBJS)
   endif
 
 instdyn: $(TARGETLIB) $(TARGETDLL).$(VER)
